eclipse-scout-core/src/form/fields/clipboardfield/ClipboardField.less

```diff
@@ -13,6 +13,7 @@
   padding: @text-field-padding-y @text-field-padding-x;
   #scout.user-select(text);
   white-space: pre-wrap;
+  position: static;
```

Comment:
Any explanation why this works? position: static will be overridden by position: relative, so it actually has no effect.